### PR TITLE
feat(pci.ai): removed home and registry tabs for ai training

### PR DESCRIPTION
**Pull request by abullet33**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/datatr-296-ai-dashboard-1.1
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #datatr-253 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description
Remove Home and Registry tabs from ai training to standardize AI tools. Registry will be moved to AI dashboard section in an other PR. 

## Related
DATATR-232 | Main branch : https://github.com/ovh/manager/tree/feat/datatr-296-ai-dashboard-1.1
- [DATATR-251](https://github.com/ovh/manager/pull/9405)
- [DATATR-204](https://github.com/ovh/manager/pull/9446)
- [DATATR-300](https://github.com/ovh/manager/pull/9454)

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/training/translations/Messages_de_DE.json
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/training/translations/Messages_de_DE.json:1]: **nitpick:** Can you please get these changes from trans team?
As per the contribution guide, developers are allowed to modify only `fr_FR` files. You can find it [here](https://github.com/ovh/manager/blob/master/CONTRIBUTING.md#translations).

File: packages/manager/modules/pci/src/projects/project/training/translations/Messages_en_GB.json
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/training/translations/Messages_en_GB.json:1]: **nitpick:** Can you please get these changes from trans team?
As per the contribution guide, developers are allowed to modify only `fr_FR` files. You can find it [here](https://github.com/ovh/manager/blob/master/CONTRIBUTING.md#translations).

File: packages/manager/modules/pci/src/projects/project/training/translations/Messages_es_ES.json
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/training/translations/Messages_es_ES.json:1]: **nitpick:** Can you please get these changes from trans team?
As per the contribution guide, developers are allowed to modify only `fr_FR` files. You can find it [here](https://github.com/ovh/manager/blob/master/CONTRIBUTING.md#translations).

File: packages/manager/modules/pci/src/projects/project/training/translations/Messages_fr_CA.json
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/training/translations/Messages_fr_CA.json:1]: **nitpick:** Can you please get these changes from trans team?
As per the contribution guide, developers are allowed to modify only `fr_FR` files. You can find it [here](https://github.com/ovh/manager/blob/master/CONTRIBUTING.md#translations).

File: packages/manager/modules/pci/src/projects/project/training/translations/Messages_it_IT.json
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/training/translations/Messages_it_IT.json:1]: **nitpick:** Can you please get these changes from trans team?
As per the contribution guide, developers are allowed to modify only `fr_FR` files. You can find it [here](https://github.com/ovh/manager/blob/master/CONTRIBUTING.md#translations).

File: packages/manager/modules/pci/src/projects/project/training/translations/Messages_pl_PL.json
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/training/translations/Messages_pl_PL.json:1]: **nitpick:** Can you please get these changes from trans team?
As per the contribution guide, developers are allowed to modify only `fr_FR` files. You can find it [here](https://github.com/ovh/manager/blob/master/CONTRIBUTING.md#translations).

File: packages/manager/modules/pci/src/projects/project/training/translations/Messages_pt_PT.json
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/training/translations/Messages_pt_PT.json:1]: **nitpick:** Can you please get these changes from trans team?
As per the contribution guide, developers are allowed to modify only `fr_FR` files. You can find it [here](https://github.com/ovh/manager/blob/master/CONTRIBUTING.md#translations).

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/training/translations/Messages_de_DE.json
Comment (abullet33) [reply packages/manager/modules/pci/src/projects/project/training/translations/Messages_de_DE.json:1]: Hi @anooparveti , as I deleted this folder "modules/pci/src/projects/project/training/registries/translations/", it has an impact on one translations, and the text "pci_projects_project_training_jobs_resubmit": "Relaunch" were display as : pci_projects_project_training_jobs_resubmit. As It have already been trad by traduction team, I figured that I just copy/paste each one in the different language without asking trad team to do something that they already done. I did not change anything else in terms of translation. Would it be ok for you or do you want me to go for translation ?

File: packages/manager/modules/pci/src/projects/project/training/translations/Messages_en_GB.json
Comment (abullet33) [reply packages/manager/modules/pci/src/projects/project/training/translations/Messages_en_GB.json:1]: Same as https://github.com/ovh/manager/pull/9456#discussion_r1232149416

File: packages/manager/modules/pci/src/projects/project/training/translations/Messages_es_ES.json
Comment (abullet33) [reply packages/manager/modules/pci/src/projects/project/training/translations/Messages_es_ES.json:1]: https://github.com/ovh/manager/pull/9456#discussion_r1232149416

File: packages/manager/modules/pci/src/projects/project/training/translations/Messages_fr_CA.json
Comment (abullet33) [reply packages/manager/modules/pci/src/projects/project/training/translations/Messages_fr_CA.json:1]: Same as https://github.com/ovh/manager/pull/9456#discussion_r1232149416

File: packages/manager/modules/pci/src/projects/project/training/translations/Messages_it_IT.json
Comment (abullet33) [reply packages/manager/modules/pci/src/projects/project/training/translations/Messages_it_IT.json:1]: Same as https://github.com/ovh/manager/pull/9456#discussion_r1232149416

File: packages/manager/modules/pci/src/projects/project/training/translations/Messages_pl_PL.json
Comment (abullet33) [reply packages/manager/modules/pci/src/projects/project/training/translations/Messages_pl_PL.json:1]: Same as https://github.com/ovh/manager/pull/9456#discussion_r1232149416

File: packages/manager/modules/pci/src/projects/project/training/translations/Messages_pt_PT.json
Comment (abullet33) [reply packages/manager/modules/pci/src/projects/project/training/translations/Messages_pt_PT.json:1]: Same as https://github.com/ovh/manager/pull/9456#discussion_r1232149416

File: packages/manager/modules/pci/src/projects/project/training/translations/Messages_de_DE.json
Comment (anooparveti) [reply packages/manager/modules/pci/src/projects/project/training/translations/Messages_de_DE.json:1]: Hi @abullet33, could you please go through the trans for the change?

Comment (abullet33) [reply packages/manager/modules/pci/src/projects/project/training/translations/Messages_de_DE.json:1]: Hi @anooparveti , yes, I will

**Review by anooparveti — APPROVED** [review]

**Pull request merged**